### DISCUSSION TITLE
Webpack: Localhost stopped working.

### DISCUSSION THREAD

**Discussion by chiestroud** [nss-evening-cohort-14/evening-client-side#101] [Help Me!] [answered]
My name is Chie Stroud.
- I did not see any other ticket for this problem.
- My locatlhost stopped working while working on assignment. I tried ctrl + c and restarted with npm start. It looks like it is working in my terminal, however, it keeps loading the page and nothing really shows up on my google chrome.
- 
![image](https://user-images.githubusercontent.com/68720317/108011850-98f36980-6fcd-11eb-9b22-cc4e73ae94dc.png)

No error message in my terminal. I tried ctrl + c / npm start multiple times but no luck so far. 

I googled these:
https://stackoverflow.com/questions/40478608/npm-start-not-starting-local-server
https://github.com/facebook/create-react-app/issues/5392
https://www.xspdf.com/resolution/52647910.html

I am a little scared to try typing random things in my terminal as this is super new to me. Thank you.

**Comment by ajawashington** [comment]
Let's hop in Study Group 1 ✨

**Comment by chiestroud** [comment, marked as answer]
Solution to this problem
1. Disable ESLint
2. Close VS Code
3. Close Teminal
4. Check VS Code and Teminal are not running in the background by opening Task Manager.
5. Open Terminal
6. Open VS Code
7. Enable ES Lint
8. npm start

Last but not least for Windows users - Please be careful of what you are "Ctrl + Z"ing.
